.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish on tag
+on:
+  release:
+    types: [published]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+jobs:
+  publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: ⬇️ Set up code
+        uses: actions/checkout@v3
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.7
+
+      - name: 👊 Bump version
+        run: |
+          TAG=$(gh release view --json tagName --jq ".tagName")
+          sed -i "s/__version__ = .*$/__version__ = '$TAG',/" src/supervisor/version.py
```

Comment:
Have you tried this on a linux machine? I got this when I tried it on my mac:
![image](https://user-images.githubusercontent.com/1326248/220440641-2e9628ec-72d7-445d-9834-ceaf78fb33a0.png)

Reply:
mac vs ubuntu sed are different

Reply:
What would the package do? Find and replace text in a file? Understand the conventions of a language and use that languages convention?

I'm not that interested in building or maintaining that. Different [ecosystems](https://github.com/agrc/api-client/blob/main/.github/workflows/release.yml#L65-L70) handle and solve the versioning problem differently. I think this is only an issue for us with python right now. I'd be more interested in spending time evaluating if [poetry](https://python-poetry.org/) would be a better tool for us. 

Creating a wrapper around sed seems like more work than it's worth.

Reply:
To be honest, I just checked the sed with https://sed.js.org/. It hadn't re-clicked in my head that this was using ubuntu. I think I've got a WSL install of ubuntu that I can test it on.

Reply:
I was just thinking that this will be a problem with our other python projects as well. I was just thinking that there's a find and replace text action already out there that would be easier than the sed command.